iothub/device/src/ClientOptions.cs

```diff
@@ -29,5 +30,27 @@ public class ClientOptions
         /// The default behavior is that <see cref="Message.MessageId"/> is set only by the user.
         /// </summary>
         public SdkAssignsMessageId SdkAssignsMessageId { get; set; } = SdkAssignsMessageId.Never;
+
+        /// <summary>
+        /// The suggested time to live value for tokens generated for SAS authenticated clients.
+        /// If unset the generated SAS tokens will be valid for 1 hour.
+        /// </summary>
+        /// <remarks>
+        /// This is used only for SAS token authenticated clients through either the
+        /// <see cref="DeviceClient.CreateFromConnectionString(string, ClientOptions)"/>/ <see cref="ModuleClient.CreateFromConnectionString(string, ClientOptions)"/> flow
+        /// or the <see cref="ModuleClient.CreateFromEnvironmentAsync(ClientOptions)"/> flow.
+        /// </remarks>
+        public TimeSpan SasTokenTimeToLive { get; set; }
+
+        /// <summary>
+        /// The time buffer before expiry when the token should be renewed, expressed as a percentage of the time to live.
```

Comment:
When I see percentage, I think it could either be 0.15 or 15. I see the type is an int, so I guess it is the latter. Still, it would be good to show an example (e.g. a value of 85 for 85%)

Reply:
Sure thing, I'll add an example.